README.md

```diff
@@ -1,7 +1,9 @@
 # Generation of SLSA3+ provenance for Go binaries
 This repository contains a reference implementation for generating non-forgeable [SLSA provenance](https://slsa.dev/) that meets the requirement for the [SLSA level 3 and above](https://slsa.dev/spec/v0.1/levels) for projects using the Go programming language.
 
-This repository contains the code, examples and technical design for our blog post on [Non forgeable SLSA provenance using GitHub workflows](TODO).
+This repository contains the code, examples and technical design for our blog post on [Non forgeable SLSA provenance using GitHub workflows](https://security.googleblog.com/2022/04/improving-software-supply-chain.html).
+
+***Note: we are working on an official release, so the code is subject to changes until then*** 
```

Comment:
More like

This is a beta release and we are looking for your feedback. The official 1.0 release should come out in the next few weeks.